build/scripts/Copy-ContextMenuResourcesToCascadiaPackage.ps1

```diff
@@ -29,6 +30,19 @@ ForEach ($pair in $Languages.GetEnumerator()) {
 		$newXml.root.data | % {
 			$null = $existingXml.root.AppendChild($existingXml.ImportNode($_, $true))
 		}
-		$existingXml.Save($PreexistingResw.FullName)
+		$XmlDocument = $existingXml # (which has been updated)
 	}
+
+	# Reset paths to be absolute (for .NET)
+	$LanguageDir = (Get-Item $LanguageDir).FullName
+	$ResPath = "$LanguageDir/Resources.resw"
+	# Force the "new" and "preexisting" paths to serialize with XmlWriter,
+	# to ensure consistency.
+	$writerSettings = [System.Xml.XmlWriterSettings]::new()
+	$writerSettings.NewLineChars = "`r`n"
+	$writerSettings.Indent = $true
+	$writer = [System.Xml.XmlWriter]::Create($ResPath, $writerSettings)
+	$XmlDocument.Save($writer)
+	$writer.Flush()
+	$writer.Close()

```

Comment:
`$XmlDocument.Save($ResPath)`

Reply:
Ah, I didn't even notice that.
This may be worth a comment IMO. When I read "consistency" I was wondering "_what_ consistency". 😅